src/forms/BoundNumberField.test.tsx

```diff
@@ -34,9 +34,9 @@ describe("BoundNumberField", () => {
     // When the user focuses
     fireEvent.focus(heightInInches());
     // And types an invalid, WIP value
-    fireEvent.input(heightInInches(), { target: { value: "11b" } });
+    fireEvent.input(heightInInches(), { target: { value: "11," } });
```

Comment:
NumberField does not allow a user to type in non-numeric characters, such as "b". Instead, changing this to another NaN value, but with an approved NumberField character.